docs/linting/CONFIGURATIONS.mdx

```diff
@@ -82,41 +80,57 @@ Additional strict rules that can also catch bugs but are more opinionated than r
 }
 ```
 
-:::tip
+:::caution
 We recommend a TypeScript project extend from this configuration only if a nontrivial percentage of its developers are highly proficient in TypeScript.
 :::
 
-## Overriding Configurations
+<configuration-contents name="strict" />
 
-These configurations are our recommended starting points, but **you don't need to use them as-is**.
-ESLint allows you to configure your own rule settings on top of any extended configurations.
-See [ESLint's Configuring Rules docs](https://eslint.org/docs/user-guide/configuring/rules#using-configuration-files).
+## Other Configurations
 
-### Suggesting Configuration Changes
+TypeScript ESLint includes a scattering of utility configurations used by the recommended configurations.
+We don't recommend using these directly; instead, extend from an earlier recommended rule.
 
-If you feel strongly that a specific rule should (or should not) be one of these configurations, please [file an issue](https://github.com/typescript-eslint/typescript-eslint/issues/new?assignees=&labels=package%3A+eslint-plugin%2Cpreset+config+change%2Ctriage&template=09-config-change.yaml&title=Configs%3A+%3Ca+short+description+of+my+proposal%3E) along with a **detailed** argument explaining your reasoning.
+### `all`
+
+Enables each the rules provided as a part of TypeScript ESLint.
+Note that many rules are not applicable in all codebases, or are meant to be configured.
```

Comment:
might be worth mentioning that there is a chance of this having conflicts because it literally is all rules with default options.